docs/user-guide/logging-configuration.md

```diff
@@ -0,0 +1,54 @@
+# Logging configuration
+
+For more information, see [Logging Utility](../extend/extend-desktop/mvd-logutility.md).
+
+## Enabling tracing
+
+To obtain more information about how a server is working, you can enable tracing within the Zowe configuration file via *components.app-server.logLevels* or *components.zss.logLevels* variable. For more information on all loggers, check out the [Extended documentation](../extend/extend-desktop/mvd-core-loggers.md).
+
+For example:
+
+```
```

Comment:
I would suggest to provide full examples with `yaml` syntax:
````
```yaml
components:
  app-server:
    logLevels:
      _zsf.routing: 0
      _zsf.install: 0
```

```yaml
components:
  zss:
    logLevels:
      _zss.traceLevel: 0
      _zss.fileTrace: 1
```
````